index.js

```diff
@@ -18,6 +20,18 @@ module.exports = function(el) {
 
   var candidates = el.querySelectorAll(candidateSelectors);
 
+  if (options.includeContainer) {
+    var matches = Element.prototype.matches || Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
+
+    if (
+      candidateSelectors.some(function(candidateSelector) {
+        return matches.call(el, candidateSelector);
+      })
+    ) {
+      candidates.unshift(el);
```

Comment:
Wouldn't we want to push this directly into `orderedTabbables`?

Reply:
Well, for my use case - no. I wanted to apply same rules to container. So if it has tabindex -1 or disabled or hidden it should not get in resulted list.

(I see the problem though that querySelectorAll returns not array, but a NodeList which does not have methods to add item to it)

Reply:
And if it has tabindex = 0, then if we push it into `orderedTabbables` then it will appear after all `basicTabbables` but before any other `orderedTabbables`, which is incorrect result.